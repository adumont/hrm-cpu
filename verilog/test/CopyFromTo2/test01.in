50 20
20 df
03 41
04 b3
03 e7
42 c4
29 0c
01 e4
12 8e
35 75
22 8a
10 70
12 91
11 87
40 3b
19 cf
42 52
25 b5
31 c7
35 d8
06 18
10 a0
49 26
13 51
26 da
12 56
29 c9
46 ce
30 71
54 bf
45 77
45 34
38 05
59 c4
23 93
32 56
24 9d
11 9a
36 cb
58 c8
22 05
01 8e
54 f9
28 36
25 ea
08 e1
59 06
10 61
36 ce
