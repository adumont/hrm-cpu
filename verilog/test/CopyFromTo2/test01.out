20
df
41
b3
e7
c4
0c
e4
8e
75
8a
70
91
87
3b
cf
52
b5
c7
d8
18
a0
26
51
da
56
c9
ce
71
bf
77
34
05
c4
93
56
9d
9a
cb
c8
